### ISSUE TITLE
Could not find com.jetbrains.intellij.java:java-compiler-ant-tasks:183.5429.37

### ISSUE THREAD

**Issue by goodandrewsoft** [JetBrains/gradle-intellij-plugin#369]
```gradle
intellij {
    version '183.5429.37'
    sandboxDirectory = "C:/Users/Andrey/.CLion2018.3"
    localPath 'F:/Run/Lprogr/JetBrains/CLion/'
    pluginName 'Helper'
    alternativeIdePath 'F:/Run/Lprogr/JetBrains/CLion/'
    downloadSources false
    type 'CL'
}
```
```
BUILD FAILED in 1s
4 actionable tasks: 1 executed, 3 up-to-date
Could not find com.jetbrains.intellij.java:java-compiler-ant-tasks:183.5429.37.
Searched in the following locations:
  - file:/F:/Run/Lprogr/JetBrains/CLion/com.jetbrains.intellij.java/java-compiler-ant-tasks/183.5429.37/ivy-183.5429.37.xml
  - file:/F:/Run/Lprogr/JetBrains/CLion/java-compiler-ant-tasks-183.5429.37-withKotlin.xml
  - file:/F:/Run/Lprogr/JetBrains/CLion/com.jetbrains.intellij.java/java-compiler-ant-tasks/183.5429.37/java-compiler-ant-tasks-183.5429.37.jar
  - file:/F:/Run/Lprogr/JetBrains/CLion/java-compiler-ant-tasks.jar
  - https://cache-redirector.jetbrains.com/plugins.jetbrains.com/maven/com/jetbrains/intellij/java/java-compiler-ant-tasks/183.5429.37/java-compiler-ant-tasks-183.5429.37.pom
  - https://cache-redirector.jetbrains.com/plugins.jetbrains.com/maven/com/jetbrains/intellij/java/java-compiler-ant-tasks/183.5429.37/java-compiler-ant-tasks-183.5429.37.jar
  - https://cache-redirector.jetbrains.com/jetbrains.bintray.com/intellij-third-party-dependencies/com/jetbrains/intellij/java/java-compiler-ant-tasks/183.5429.37/java-compiler-ant-tasks-183.5429.37.pom
  - https://cache-redirector.jetbrains.com/jetbrains.bintray.com/intellij-third-party-dependencies/com/jetbrains/intellij/java/java-compiler-ant-tasks/183.5429.37/java-compiler-ant-tasks-183.5429.37.jar
  - https://cache-redirector.jetbrains.com/repo1.maven.org/maven2/com/jetbrains/intellij/java/java-compiler-ant-tasks/183.5429.37/java-compiler-ant-tasks-183.5429.37.pom
  - https://cache-redirector.jetbrains.com/repo1.maven.org/maven2/com/jetbrains/intellij/java/java-compiler-ant-tasks/183.5429.37/java-compiler-ant-tasks-183.5429.37.jar
Required by:
    project :
10:11:39: Task execution finished 'verifyPlugin'.
```

F:/Run/Lprogr/JetBrains/CLion/ = CL-183.5429.37

How i can **MANUALLY** set `java:java-compiler-ant-tasks`?

**Comment by zolotov** [comment]
Looks like a but. Could you try to add an additional maven repo to your script, please? Does it help?

```
repositories {
    maven {
        url 'https://www.jetbrains.com/intellij-repository/releases'
    }
}
```

> How i can **MANUALLY** set `java:java-compiler-ant-tasks`?

https://github.com/JetBrains/gradle-intellij-plugin#instrumenting-dsl

**Comment by goodandrewsoft** [comment]
If i remove `localPath 'F:/Run/Lprogr/JetBrains/CLion/'` all fine, but downloads additional not needed (**~1800 mb**) distribs to my `.gradle` folder. 

Full [build.gradle](https://gist.github.com/goodandrewsoft/dbbdd61a147bb577f089209ce0d5ad17):
Same error!

Please edit my `build.gradle` to work without needing to download gigabytes of data.

**Comment by zolotov** [comment]
@goodandrewsoft I didn't ask you to remove `localPath`. Have you tried to add a repository?

**Comment by goodandrewsoft** [comment]
Yes, see full build.gradle:

**Comment by goodandrewsoft** [comment]
Yes, see full build.gradle:

**Comment by zolotov** [comment]
Maybe this could help?

```
buildscript {
  repositories {
      maven {
          url 'https://www.jetbrains.com/intellij-repository/releases'
      }
  }
}
```

**Comment by goodandrewsoft** [comment]
Вы издеваетесь ?:)
![image](https://user-images.githubusercontent.com/2471892/53566997-8b394500-3b66-11e9-8027-9a887c3e61c5.png)

**Comment by zolotov** [comment]
Нет, я не издеваюсь, я пытаюсь вам помочь хоть что-то собрать без необходимости выкачивать дистрибутив CLion или ожидания новой версии gradle-intellij-plugin.
Тогда попробуйте то, что я просил попробовать изначально: https://github.com/JetBrains/gradle-intellij-plugin/issues/369#issuecomment-468215368

**Comment by goodandrewsoft** [comment]
Исправьте мой **гист** или не отвечайте вовсе. Осталось домой к вам придти с компом и показать что я это уже все прописывал. ....

**Comment by zolotov** [comment]
@goodandrewsoft it's enough to show the build.gradle with the suggested changes and the output. 

Neither gist nor video doesn't contain these:

```
repositories {
      maven {
          url 'https://www.jetbrains.com/intellij-repository/releases'
      }
  }
```

Only this

```
buildscript {
  repositories {
      maven {
          url 'https://www.jetbrains.com/intellij-repository/releases'
      }
  }
}
```

And please don't be rude, I'm actually trying to find a workaround for you but instead of applying suggested changes you demanding some actions from me using not really acceptable tone. 

If you don't need a workaround, I'm completely fine with this, you can wait for the next version where this bug will be likely fixed. Still, it's not the reason to talk like that.

**Comment by goodandrewsoft** [comment]
![image](https://user-images.githubusercontent.com/2471892/53568876-c4c07f00-3b6b-11e9-8e3c-8211b806aab6.png)
Мы можем так до вечера!

**Comment by zolotov** [comment]
`repositories` block must be after `plugins` block. It's a gradle restriction and the error message quite descriptive

**Comment by goodandrewsoft** [comment]
I thinked, this

```
repositories {
      maven {
          url 'https://www.jetbrains.com/intellij-repository/releases'
      }
  }

```
and this
```
buildscript {
  repositories {
      maven {
          url 'https://www.jetbrains.com/intellij-repository/releases'
      }
  }
}
```
the same.
Now its worked. Thanks.!

**Comment by AlexeyKorshun** [comment]
Hi, I try to use your plugin, but I have an issue ` Could not find com.jetbrains.intellij.java:java-compiler-ant-tasks:203.7717.56.2031.7583922.` . Before upgrade my IDEA to new version, it worked fine. can you help, what can be wrong?

**Comment by ant-druha** [comment]
Same issue reported https://stackoverflow.com/q/68655763/2000323 cc @hsz

**Comment by zolotov** [comment]
@AlexeyKorshun as you use local android studio (not IDEA) as an SDK, you have [to specify a particular compiler version](https://github.com/JetBrains/gradle-intellij-plugin#instrumenting-dsl) that should be used for instrumenting.

There is no compiler with `203.7717.56.2031.7583922` version. All available versions are listed here: https://www.jetbrains.com/intellij-repository/releases. In your case, I think 203.7717.56 would be just fine.

So the configuration should look like this:

```
instrumentCode {
  compilerVersion = "203.7707.56"
}
```

**Comment by AlexeyKorshun** [comment]
@zolotov why do you think I use AS as an SDK? 

but ok, first of all, thx for plugin :)

and current version is 0.7.2 in project. I build plugin in IDEA for IC type, as I know it is IDEA.

**Comment by AlexeyKorshun** [comment]
sorry I find where I can change a version of a compiler, but right now I have an issue 
`Could not find com.jetbrains.intellij.idea:ideaIC:203.7707.56.`

**Comment by zolotov** [comment]
> @zolotov why do you think I use AS as an SDK?

I suggest that from the version: 203.7717.56.2031.7583922. It's impossible version for IDEA IC.

If it's not, please share the entire build.gradle

**Comment by AlexeyKorshun** [comment]
my sorry, thx for find a problem. I change my version. It was a true

**Comment by hsz** [comment]
@CoXier  This has been already addressed with a2f3837dad801c15a20243f6998d4d9a99eb336d
